src/App.js

```diff
@@ -1,10 +1,30 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes,
+} from 'react-router-dom';
 import './App.css';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
+import './footer.scss';
 
 function App() {
   return (
     <>
-      <Calculator />
+      <Router>
+        <div>
+          <Routes>
+            <Route path="/calculator" element={<Calculator />} />
+            <Route path="/quotes" element={<Quotes />} />
+            <Route path="/" element={<Home />} />
+          </Routes>
+        </div>
+      </Router>
+      <footer className="footer">
+        <p>© 2023 Math Magicians</p>
+      </footer>
```

Comment:
- [ ] [Optional] kindly make the footer stick to the bottom of the page, this will give a good UI
![Screenshot from 2023-02-28 21-07-10](https://user-images.githubusercontent.com/66097371/221967121-2d9a278c-9e34-457a-85a1-ab7f093dea3d.png)